airflow-core/pyproject.toml

```diff
@@ -134,7 +134,7 @@ dependencies = [
     # Does not work with it Tracked in https://github.com/fsspec/universal_pathlib/issues/276
     "universal-pathlib>=0.2.2,!=0.2.4",
     "uuid6>=2024.7.10",
-    "apache-airflow-task-sdk<1.1.0,>=1.0.0",
+    "apache-airflow-task-sdk<=1.1.0,>=1.0.0",
```

Comment:
BTW. Why 1.1.0 and not 1.0.1 - are we sure there will be no bug fix release of task -sdk?

Reply:
I think we should propose some versioning strategy we should apply for it.

Reply:
Main should always point to minor.

But fixes will be cherry-picked

Reply:
Ah. Ok . So I guess  we will keep it synced -> Airflow 3.0.0 -> Task Sdk 1.0.0, Airlfow 3.1 -> Task SDK 1.1 (at least for now until we do not fully separate these and finalize the move ? 

Because after that - we could simply start relesing task-sdk from main, similarly as we do with providers. No need to cherry-pick stuff to v3-0-test assuming that task-sdk will be (should be) forward compatible.?

In this case the upper bound here should be `< 1.2.0` actually - because 1.2.0 is likely the version when we change the approach of task-sdk releasing

Reply:
Also lower-binding should be >= 1.1.0 in this case

Reply:
Good call

Reply:
Updated